docs/netdata-agent/configuration/dynamic-configuration.md

```diff
@@ -9,7 +9,9 @@ The Dynamic Configuration Manager allows direct configuration of collectors and
 
 **Cloud Connection and Security**: Nodes using Dynamic Configuration Manager require a connection to Netdata Cloud. This ensures proper permission handling and data security.
 
-**Permissions**: Admins and Managers can view, edit, and submit configurations within the UI.
+> **Info**
+>
+> For details on what **permissions** apply based on a user's role, please refer to the [Role Based Access documentation](/docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md#dynamic-configuration-manager) 
```

Comment:
1. I noticed that for some reason you often try to put some information in blockquotes. Why? You should not do this with important information.
2. I don't think we need to use "please" in our docs, should be more formal.

Reply:
1. It was some time ago that we said notes and info looked better in blockquotes because they stand out in learn more than bold text, especially in that section, there is a lot of bold text.
2. Agree, I will remove it